gradle.properties

```
@@ -2,7 +2,7 @@ org.gradle.caching=true
 group=io.ballerina.stdlib
 version=1.5.0-SNAPSHOT
 #dependency versions
-ballerinaLangVersion=2201.2.0
+ballerinaLangVersion=2201.3.0
```

```suggestion
ballerinaLangVersion=2201.3.0-20221004-103700-89b78c16
```
We have to use the timestamped version, as we have not released 2201.3.0 yet

---

Should be updated in now